### PR TITLE
Home and basic folder structure

### PR DESCRIPTION
**Pull request by tanveertkd**
This PR contains the homepage and outlines the basic folder structure of the project.
Live link included below or click [here](https://deploy-preview-1--laughing-pare-6939f2.netlify.app/)

### CODE REVIEW

File: src/components/Showcase/Showcase.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import './showcase.css';
+
+const Showcase = () => {
+
+    const {categoryItems, setCategoryItems} = useCategories();
+    console.log("from showcase.jsx", categoryItems)
+    useEffect(()=>{
+        (async ()=>{
+            const { data: {categories} } = await axios.get("./api/categories")
+            console.log("showcase from showcase.jsx", categories)
```
Comment (msdeshmukh009) [inline src/components/Showcase/Showcase.jsx:14]: You can remove this console.

**Review by msdeshmukh009 — APPROVED** [review]
The code is looking good.
The footer is looking really professional.
Good work!!

**Review by manojasarsa — APPROVED** [review]
The code looks clean. 
Also, the home page is looking neat and cool.
Good to merge!

**Review by mimansha-swarup — COMMENTED** [review]
code looks clean and UI is also minimal 
but for screen sizes 320 to 425px hero's image is overflowing 
image
and  there is no space between hero image and  shop now button

File: src/components/Showcase/Showcase.jsx (hunk 1 above)
Comment (tanveertkd) [reply src/components/Showcase/Showcase.jsx:14]: Will do. Thanks!

**Comment by tanveertkd** [comment]
> 

I'm actually yet to make it properly responsive. But I'll do it asap. Thanks!

**Pull request merged**